docs/README.md

```diff
@@ -1,13 +1,15 @@
 <h1><a href='https://discord.com'><img src='../icons/app.png' width='64px'></a> Electron Discord Web App </h1>
-
-[![Build](https://github.com/SpacingBat3/electron-discord-webapp/actions/workflows/build.yml/badge.svg)](https://github.com/SpacingBat3/electron-discord-webapp/actions/workflows/build.yml)
-[![MIT license](https://img.shields.io/badge/License-MIT-C23939.svg)](COPYING)
-[![Electron](https://img.shields.io/badge/Made%20with-Electron-486F8F.svg)](https://www.electronjs.org/)
-[![GitHub release](https://img.shields.io/github/release/SpacingBat3/electron-discord-webapp.svg)](../../../tags)
-[![Github downloads](https://img.shields.io/github/downloads/SpacingBat3/electron-discord-webapp/total.svg)](../../../releases)
-[![GitHub contributors](https://img.shields.io/github/contributors/SpacingBat3/electron-discord-webapp.svg)](../../../graphs/contributors)
-[![PRs/Translations Welcome](https://img.shields.io/badge/PRs/Translations-welcome-brightgreen.svg)](#want-to-contribute-to-my-project)
-[![Pi-Apps badge](https://badgen.net/badge/Pi-Apps%3F/Yes!/c51a4a?icon=https://gitcdn.link/repo/Botspot/pi-apps/master/icons/logo.svg)](https://github.com/Botspot/pi-apps)
+<p align="center">
+  
+  [![Build](https://github.com/SpacingBat3/electron-discord-webapp/actions/workflows/build.yml/badge.svg)](https://github.com/SpacingBat3/electron-discord-webapp/actions/workflows/build.yml)
+  [![MIT license](https://img.shields.io/badge/License-MIT-C23939.svg)](COPYING)
+  [![Electron](https://img.shields.io/badge/Made%20with-Electron-486F8F.svg)](https://www.electronjs.org/)
+  [![GitHub release](https://img.shields.io/github/release/SpacingBat3/electron-discord-webapp.svg)](../../../tags)
+  [![Github downloads](https://img.shields.io/github/downloads/SpacingBat3/electron-discord-webapp/total.svg)](../../../releases)
+  [![GitHub contributors](https://img.shields.io/github/contributors/SpacingBat3/electron-discord-webapp.svg)](../../../graphs/contributors)
+  [![PRs/Translations Welcome](https://img.shields.io/badge/PRs/Translations-welcome-brightgreen.svg)](#want-to-contribute-to-my-project)
+  [![Pi-Apps badge](https://badgen.net/badge/Pi-Apps%3F/Yes!/c51a4a?icon=https://gitcdn.link/repo/Botspot/pi-apps/master/icons/logo.svg)](https://github.com/Botspot/pi-apps)
+</p>
```

Comment:
Just in case, GitHub's HTML support is quite limited – not sure if the aligment really works there. I would be suprised if so.

The one thing I'm sure is that you can't style much it, so better don't try adding a `<style>` to it.

Reply:
I'm using html to center it

Reply:
alignment works, I did it in the README of one of my projects.

Reply:
Oh, I it works!

I'll give it a try then on my own...

Might need to close this pull request, as I believe it has a conflict with the current branch + I changed a lot about the badges.

Reply:
@SpacingBat3 it only works when I using html tags to add the badges.